src/components/Login.svelte

```diff
@@ -59,31 +75,54 @@
   input.error {
     border: rgb(255, 0, 0) 1px solid;
   }
+
+  .flex {
+    display: flex;
+    justify-content: center;
+  }
+
+  .sign-up {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
Does svelte recommend putting javascript, CSS, and html all in one file? Traditionally, it's better if these three are separated for easier readability / logic separation. For example, if you're only making a Javascript change, you won't need to even look at CSS/HTML.

If svelte doesn't have a preference, I would recommend separating the three

Reply:
Yes, this is a svelte preference :)